app/models/miq_report_result.rb

```diff
@@ -167,24 +167,13 @@ def purge_for_user
   end
 
   def to_pdf
-    page_size = "a4"
-    if report.rpt_options && report.rpt_options[:pdf]
-      page_size = report.rpt_options[:pdf][:page_size] || "a4"
-    end
-
-    curr_tz = Time.zone # Save current time zone setting
-    user = userid.include?("|") ? nil : User.find_by_userid(userid)
-    Time.zone = user ? user.get_timezone : MiqServer.my_server.server_timezone
```

Comment:
To elaborate a bit, the only reason I could see for doing this was to pass `Time.zone` into the `format_timezone` helper - why not just pass in whatever we were setting `Time.zone` to? I don't see anything else doing anything with the system timezone?